src/Views/batchprocessor.html

```diff
@@ -135,27 +140,31 @@ <h3>Select Basin Characteristics</h3>
                     <span ng-if="vm.parametersAllChecked">Select All Basin Characteristics</span>
                     <span ng-if="!vm.parametersAllChecked">Unselect All Basin Characteristics</span>
                 </button>
+
                 <!-- create table of parameters checkboxes -->
-                <table class="table table-condensed">
-                    <thead class="param-table">
-                        <tr>
-                            <th class="col-xs-2">Select</th>
-                            <th class="col-xs-3">Parameter</th>
-                            <th class="col-xs-7">Description</th>
-                        </tr>
-                    </thead>
-                    <tbody ng-repeat="parameter in vm.availableParamList">
-                        <tr>
-                            <td>
-                                <checkbox large ng-model="parameter.checked"
-                                    ng-change="vm.updateSelectedParamList(parameter)" ng-disabled="!parameter.toggleable">
-                                </checkbox>
-                            </td>
-                            <td><span>&nbsp;&nbsp;{{parameter.code}}</span></td>
-                            <td><span>&nbsp;&nbsp;{{parameter.description}}</span></td>
-                        </tr>
-                    </tbody>
-                </table>
+                <div ng-hide="!vm.parametersListSpinner"><i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i></div>

```

Comment:
Can you please add a little space between the the blue button and the spinner?

Reply:
Done!